src/box/lua/net_box.lua

```diff
@@ -1216,6 +1233,18 @@ function remote_methods:ping(opts)
     return (pcall(self._request, self, 'ping', opts))
 end
 
+function remote_methods:shutdown(opts)
+    check_remote_arg(self, 'shutdown')
+    return (pcall(self._request, self, 'shutdown', opts))
+end
+
+function remote_methods:set_shutdown_handler(handler)
+    if type(handler) ~= 'function' then
```

Comment:
You need use check_remote_arg(self, 'set_shutdown_handler') first of all

Reply:
Fixed